Source/Core/Core/HW/SI/SI_Device.h

```diff
@@ -44,6 +44,7 @@ enum TSIDevices
   SI_GBA = 0x00040000,
   SI_GC_CONTROLLER = (SI_TYPE_GC | SI_GC_STANDARD),
   SI_GC_KEYBOARD = (SI_TYPE_GC | 0x00200000),
+  SI_GC_PEDOMETER = 0x08020000,
```

Comment:
What about using this instead (for consistency with the other GC members)?
```suggestion
  SI_GC_PEDOMETER = (SI_TYPE_GC | 0x00020000),
```

Reply:
Will do!